logging/logger.ts

```diff
@@ -0,0 +1,19 @@
+/// <reference types="node" />
+import winston from 'winston';
```

Comment:
```suggestion
import winston from 'winston';
```
You can also switch off this rule `import/no-extraneous-dependencies`